configs/mainnet.yaml

```diff
@@ -81,7 +77,6 @@ MIN_PER_EPOCH_CHURN_LIMIT: 4
 # 2**16 (= 65,536)
 CHURN_LIMIT_QUOTIENT: 65536
 
-
```

Comment:
Unnecessary polluting diff styling `configs/mainnet.yaml` and `configs/minimal.yaml`, should be reverted for a cleaner PR.